fix: Sidecar restart spawning ReplicaSet every time when ReplicaSet did not finish progressing (#1444)
kyma-project/istio: docs/release-notes/1.17.3.md

## Fixed Bugs

> - We've fixed a bug that was causing the Istio operator to create a new ReplicaSet each time a reconciliation of a failing, not up-to-date Pod with sidecar happened. [#1444](https://github.com/kyma-project/istio/pull/1444)

Reviewer: ```suggestion
- We've fixed a bug that caused Istio Operator to create a new ReplicaSet during each reconciliation of a failing, outdated Pod with a sidecar. [#1444](https://github.com/kyma-project/istio/pull/1444)
```
